addons/l10n_de_sale/models/sale.py

```diff
@@ -28,6 +28,7 @@ def _compute_l10n_de_template_data(self):
                 data.append((_('Customer Reference'), record.client_order_ref))
             if record.user_id:
                 data.append((_("Salesperson"), record.user_id.name))
+            data += record._get_additional_template_data()
```

Comment:
```
if 'incoterm' in record._fields:
    data.append(...)
```
Open debate, I recognize it's less clean, but less code to maintain :D.
(less clean, but less code to maintain :thinking:)

Reply:
As agr pointed out, we usually go through a dummy method so I would tend to prefer this method since it is cleaner as well.
But I agree that your solution would greatly reduce the code to maintain :thinking: 

I think that wan's opinion would be useful here, as he seems to manage this part (or at least he was the one who wrote this). 

Hello @william-andre , can we get our opinion here please? :slightly_smiling_face:

Reply:
Yep, you can use `if 'incoterm' in record._fields:`, this is a hacky module already anyways

Reply:
@william-andre Thanks!
I just pushed changes according to this :slightly_smiling_face: